sway/tree/output.c

```diff
@@ -390,6 +391,33 @@ void output_get_box(struct sway_output *output, struct wlr_box *box) {
 	box->height = output->height;
 }
 
+static void handle_destroy_non_desktop(struct wl_listener *listener, void *data) {
+	struct sway_output_non_desktop *output =
+		wl_container_of(listener, output, destroy);
+
+	sway_log(SWAY_DEBUG, "Destroying non-desktop output '%s'", output->wlr_output->name);
+
```

Comment:
I noticed that we never withdraw outputs, there should be a call to `wlr_drm_lease_v1_manager_withdraw_output` here to notify the clients if the output is leased.

Reply:
Doesn't `wlr_drm_lease_v1_manager` listen for output destroy already?

Reply:
Ah yeah, nevermind then.